README.md

```diff
@@ -33,56 +33,44 @@ List them by running the following command:
 
 ### Surface pipeline arguments (optional)
-* --fstess: Use mri_tesselate instead of marching cube (default) for surface creation
-* --fsqsphere: Use FreeSurfer default instead of novel spectral spherical projection for qsphere
-* --fsaparc: Use FS aparc segmentations in addition to DL prediction (slower in this case and usually the mapped ones from the DL prediction are fine)
-* --surfreg: Create Surface-Atlas (sphere.reg) registration with FreeSurfer (for cross-subject correspondence or other mappings)
-* --parallel: Run both hemispheres in parallel
-* --threads: Set openMP and ITK threads to <int>
-* --no_fs_T1: Do not generate T1.mgz (normalized nu.mgz included in standard FreeSurfer output) and create brainmask.mgz directly from norm.mgz instead. Saves 1:30 min.
-
+* `--surf_only`: only run the surface pipeline recon_surf. The segmentation created by FastSurferCNN must already exist in this case.
+* `--fstess`: Use mri_tesselate instead of marching cube (default) for surface creation
+* `--fsqsphere`: Use FreeSurfer default instead of novel spectral spherical projection for qsphere
+* `--fsaparc`: Use FS aparc segmentations in addition to DL prediction (slower in this case and usually the mapped ones from the DL prediction are fine)
+* `--parallel`: Run both hemispheres in parallel
+* `--threads`: Set openMP and ITK threads to <int>
+* `--no_surfreg`: Skip surface registration with FreeSurfer (if only stats are needed), for the sake of speed. 
+* `--no_fs_T1`: Do not generate T1.mgz (normalized nu.mgz included in standard FreeSurfer output) and create brainmask.mgz directly from norm.mgz instead. Saves 1:30 min.
 
 ### Other
-* --py: which python version to use. Default: python3.8
-* --seg_only: only run FastSurferCNN (generate segmentation, do not run the surface pipeline)
-* --surf_only: only run the surface pipeline recon_surf. The segmentation created by FastSurferCNN must already exist in this case.
-* --seg_with_cc_only: Run FastSurferCNN (generate segmentation) and recon_surf until corpus callosum (CC) is added in (no surface models will be created in this case!)
-* --conformed_name: Name of the file in which the conformed input image will be saved. Default location: \$SUBJECTS_DIR/\$sid/mri/orig.mgz
-* --config_sag: Path to the sagittal config file
-* --config_ax: Path to the axial config file
-* --config_cor: Path to the axial coronal file
-* --ignore_fs_version: Switch on to avoid check for FreeSurfer version. Program will terminate if the supported version (see recon-surf.sh) is not sourced. Can be used for testing dev versions.
-* --help: Prints helptext out
+* `--vox_size 1|auto`:  Force processing at a specific voxel size. Only "auto" and "1" are supported values. If "1" is specified, the T1w image will be conformed to 1mm voxel size and processed, if "auto" is specified (default), the T1w image will be conformed isometric voxels of the smallest voxel edge in the image for highres processing.
```

Comment:
the vox_size option was changed. run_fastsurfer.sh has updated docs

Reply:
This is resolved. See [here](https://github.com/Deep-MI/FastSurfer/pull/231)